src/monitor.cpp

```diff
@@ -161,6 +161,28 @@ unsigned long mTriggerUpdate = 0;
 unsigned long initialMillis = millis();
 unsigned long initialTime = 0;
 
+void getTime(unsigned long* currentHours, unsigned long* currentMinutes, unsigned long* currentSeconds){
+  
+  //Check if need an NTP call to check current time
+  if((mTriggerUpdate == 0) || (millis() - mTriggerUpdate > UPDATE_PERIOD_h * 60 * 60 * 1000)){ //60 sec. * 60 min * 1000ms
+    if(WiFi.status() == WL_CONNECTED) {
+        timeClient.update(); //NTP call to get current time
+        mTriggerUpdate = millis();
+        initialTime = timeClient.getEpochTime(); // Guarda la hora inicial (en segundos desde 1970)
+        Serial.print("TimeClient NTPupdateTime ");
+    }
+  }
+
+  unsigned long elapsedTime = (millis() - mTriggerUpdate) / 1000; // Tiempo transcurrido en segundos
+  unsigned long currentTime = initialTime + elapsedTime; // La hora actual
+
+  // convierte la hora actual en horas, minutos y segundos
+  *currentHours = currentTime % 86400 / 3600;
+  *currentMinutes = currentTime % 3600 / 60;
+  *currentSeconds = currentTime % 60;
+
+}
+
 String getTime(void){
   
```

Comment:
Could you please reuse your new getTime function here and just format result as string, in order to avoid code duplication?

Reply:
hi @metalnow Roman is right, you should try to fit to clock_data structure and change current TIme on your device.
For now is ok, but try to do this change please to maintain clean code on other PR. 

Nexts devices will run same structure

Reply:
It's not possible to reuse clock data structure in this case. His clock data is lighter and he requires numbers, instead of strings to draw it correctly. Its okay I believe to have another light clock data structure with numbers. Im just saying that his new getTime function needs to be called from this to avoid code duplications.

Reply:
oh, now I understand. Right to do this on our next PR.

Reply:
Okay. I will make a new function name of gettime for preventing confusion. Thanks for the comments.